src/main/java/com/revature/lostchapterbackend/controller/BookController.java

```diff
@@ -0,0 +1,48 @@
+package com.revature.lostchapterbackend.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.lostchapterbackend.model.Book;
+import com.revature.lostchapterbackend.service.BookService;
+
+@RestController
+public class BookController {
+	
+	@Autowired
```

Comment:
BookService is not fully completed, the getBookById() method is the only method needed in the carts feature to firstly check if a book exist or not, and then add it to the cart if it exist.